server-spi/src/main/java/org/keycloak/theme/ThemeProperties.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.keycloak.theme;
+
+/**
+ * Property keys for theme.properties files.
+ */
+public final class ThemeProperties {
```

Comment:
I would avoid creating this class and move these constants to `org.keycloak.theme.Theme`. Not sure if we need another class to hold constants.

In the `Theme` class there is an enum already for each theme type. That is why I think we can just add these constants there too as they are related to a theme.

Reply:
Moved to `Theme`